db/seeds/cip_seed.rb

```diff
@@ -94,6 +94,8 @@
   ["4b0c92ea-f9ea-4794-9ebf-e651ebea4719", "2.6: Reflecting on learning", "## Time allocation\n\n10 minutes\n\n## Instructions\n\n- Use your notepad to reflect on your learning in this Block and further professional development.\n- Write just a couple of sentences for each question.\n\n$CTA\n\n<h2>In your notepad</h2>\n<ol>\n  <li>What have you learned in this Block?</li>\n  <li>What do you need to learn more about?</li>\n</ol>\n$CTA\n\n", "701a5401-d77b-48af-8c4d-3bee197896f0", "48c66862-8a65-4908-8fdc-eff63d632940", 4],
```

Comment:
I wonder why they moved. If you move them back where they were, re-seed and re-export, do they stay in their old place, or do they move back here?

Reply:
I'm not sure, i put it back to the original place in the seed, re-seeded and the seed dump moved it back to lines 97 and 98.

Reply:
Weird.

I got the old files, I updated ambition year content in seed file, I dropped my db, recreated it, seeded it, and then nothing else moved.

I _think_ it's to do with the last time an object was updated. I think `CoreInductionProgrammeExporter` orders objects by default Rails order, which might be when they were last updated. That's annoying. But since we control the queries we might be able to specify that we want the order the elements were created by to be the deciding factor. Wanna try it? 

In `CoreInductionProgrammeExporter` we would add
```
.order('created_at ASC')
```
to each of the `.all` queries. And see if it is any better :)